.storybook/main.js

```diff
@@ -27,11 +26,13 @@ module.exports = {
   },
   webpackFinal: async config => {
     // Get the specifications object and replace with a real object in the spec.ts file
+    console.log('Building specs...');
     const specs = await getSpecifications();
 
-    // modules/specifications/lib/specs.ts
+    console.log('Building component prop tables...');
```

Comment:
Any reason to keep these logs?

Reply:
I guess they only show up in the build, so maybe it's fine.

Reply:
I don't think they are necessary